bun/whatsapp-with-vonage/src/main.ts

```diff
@@ -0,0 +1,66 @@
+import jwt from "jsonwebtoken";
+import { fetch } from "undici";
+import { getStaticFiles, throwIfMissing } from "./utils.ts";
+
+export default async ({ req, res, log, error }: any) => {
+  throwIfMissing(Bun.env, [
+    "VONAGE_API_KEY",
+    "VONAGE_ACCOUNT_SECRET",
+    "VONAGE_WHATSAPP_NUMBER",
+    "VONAGE_SIGNATURE_SECRET",
+  ]);
+
+  if (req.method === "GET") {
+    return res.send(getStaticFiles("index.html"), 200, {
+      "Content-Type": "text/html; charset=utf-8",
+    });
+  }
+  const authHeader: string = (req.headers.authorization ?? "").split(" ")[1];
+  const decodedToken = jwt.verify(authHeader, Bun.env.VONAGE_SIGNATURE_SECRET, {
+    algorithms: ["HS256"],
+  });
```

Comment:
Please add a space before this section and use structure similar to here
https://github.com/appwrite/templates/blob/6dab35f1f221ba075640d93f5a8e374f4fba4349/node/whatsapp-with-vonage/src/main.js#L20

First assign `token` variable

Then assign `decoded` variable